src/TestSuite/Constraint/Email/MailContains.php

```diff
@@ -52,6 +52,32 @@ public function matches($other): bool
         return false;
     }
 
+    /**
+     * @return string
+     */
+    private function getTypeMethod(): string
```

Comment:
Why not protected?

Reply:
because the method is only called from the class itself

Reply:
We don't do that in a framework context where not absolutely required, as this also prohibits further extension where needed.

Reply:
ok I changed it